install/install.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2021 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This file is part of public API.
+ *
+ * By default, the package runs this script during the installation
+ * process unless one of the env flags is provided.
+ */
+
+// Copied form Puppeteer
```

Comment:
typo: "from"

Have you considered using `puppeteer.createBrowserFetcher(options)` instead of duplicating the underlying Browser Fetcher code (+ the additional `typescript-if-required` complexity) here? Not having to maintain this logic across multiple projects would be nice.

Another option is to write a simple standalone script that just fetches the latest binary.

Reply:
fixed typo.

If you mean bring Puppeteer as a dependency, it seems like an overkill.

The `BrowserFetcher` is a relatively simple script injecting retrieving binary path for different platforms logic, which used both during setup and server running. From that perspective it seems to make sense to share it between `install` and `server` workflows.

Reply:
> The `BrowserFetcher` is a relatively simple script injecting retrieving binary path for different platforms logic, which used both during setup and server running. From that perspective it seems to make sense to share it between `install` and `server` workflows.

We’re in agreement here — I’m not arguing against using `BrowserFetcher` in both workflows.

> If you mean bring Puppeteer as a dependency, it seems like an overkill.

Hmm, which criteria are you using to determine whether something is “overkill” or not?

This PR currently introduces 6 new `dependencies` + 3 new `devDependencies`, and duplicates the `BrowserFetcher` code from Puppeteer, which includes logic that doesn’t seem necessary here (e.g. dealing with proxy env vars). 

I’m suggesting it could instead be 1 new dependency with no code duplication/additional maintenance overhead.

The latter option sounds simpler, more maintainable, and less “overkill” to me — can you help me understand what you think is more complex about it?

Reply:
let me try it (puppeteer dependency) out